test/unit/test_sap_cli_gcts_utils.py

```
@@ -12,9 +12,13 @@
     print_gcts_message,
     dump_gcts_messages,
     gcts_exception_handler,
+    print_process_messages,
 )
 from sap.rest.errors import HTTPRequestError
```

Flake8 (7.3.0)

[error] 'sap.rest.errors.HTTPRequestError' imported but unused

(F401)